saferpayofficial.php

```diff
@@ -116,6 +116,28 @@ public function getService($service)
         return $containerProvider->getService($service);
     }
 
+    public function hookDisplayOrderConfirmation($params)
+    {
+        if (empty($params['order'])) {
+            return '';
+        }
+
+        /** @var Order $psOrder */
+        $psOrder = $params['order'];
+
+        /** @var \Invertus\SaferPay\Repository\SaferPayOrderRepository $repository */
+        $repository = $this->getService(\Invertus\SaferPay\Repository\SaferPayOrderRepository::class);
+
+        $sfOrder = $repository->getByOrderId((int) $psOrder->id);
+        if (!$sfOrder->pending) {
+            return '';
+        }
+
+        //@todo: translate and move to template if needed when requirements are clear
```

Comment:
Sitas todo lieka. Reik issiaiskint ko mes norim is to pending status pag'o. Ar uztenka translatint sita hooka ar kazka daugiau daryt.